### ISSUE TITLE
NEWBIE Question - How to deploy

### ISSUE THREAD

**Issue by mnalgirkar** [patrickrb/threejsTornado#12]
I am using your project as my template for three.js tryouts. Can you provide instructions on how to deploy the project to a website? Thanks!

**Comment by patrickrb** [comment]
Were you able to get it running locally in develop? Just trying to understand exactly what you're asking.

**Comment by mnalgirkar** [comment]
Yes, I have been able to build and run it locally. I was just trying to understand the steps to test this out on a server/ website. Presume it involves uploading the Client, Server folders and modifying the config files, but if you can add any instructions that would be great. Thank you!
